docs/docs/09-typescript/01-resources.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Resources
+id: resources
+---
+
+Resources are created similar to other CDKs (awscdk, cdktf, cdk8s, etc.). They make up the tree that represents your infrastructure.
+Resources are classes typically constructed with at least two arguments: The scope (parent) resource and an id.
+
+The first argument for `main` is the root of this tree of resources and can be used to construct your first resource:
```

Comment:
I feel like we need to explain `main` first (maybe in the topic about entrypoints?)